apps/ledger-live-desktop/tests/utils/allureUtils.ts

```diff
@@ -16,11 +17,13 @@ export async function captureArtifacts(page: Page, testInfo: TestInfo) {
   const screenshot = await page.screenshot();
   await testInfo.attach("Screenshot", { body: screenshot, contentType: "image/png" });
 
-  const speculosScreenshot = await takeScreenshot();
-  await testInfo.attach("Speculos Screenshot", {
-    body: speculosScreenshot,
-    contentType: "image/png",
-  });
+  if (IS_NOT_MOCK) {
```

Comment:
```suggestion
  if (!process.env.MOCK) {
  ...
  }
```

Reply:
It would not work the same. The IS_NOT_MOCK constant is a boolean that is true if process.env.MOCK is "0" and false otherwise. If you replace IS_NOT_MOCK with process.env.MOCK, you would be checking the value of the environment variable directly, which is a string, not a boolean.